Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2216,6 +2216,20 @@ DeprecateAESCFBWebCryptoEnabled:
     WebCore:
       default: true
 
+DeprecateCoreCommonCryptoEnabled:
```

Comment:
I think "CryptoKitEnabled" would be a better name for this.  It avoids double negatives when reading code.

Reply:
done